whois-update/src/main/java/net/ripe/db/whois/update/sso/SsoTranslator.java

```diff
@@ -54,9 +58,12 @@ public RpslAttribute translate(final String authType, final String authToken, fi
                 if (authType.equals("SSO")) {
                     if (!updateContext.hasSsoTranslationResult(authToken)) {
                         try {
-                            final String uuid = crowdClient.getUuid(authToken);
+                            LOGGER.info("getting details for {}", authToken);
```

Comment:
Log at debug?

Reply:
I added this for testing, it has been removed.